make/Makefile.container.mk

```diff
@@ -62,3 +62,34 @@ endif
 
 ## container-push: Pushes all container images to quay
 container-push: container-push-kiali-quay
+
+# Ensure "docker buildx" is available and enabled. For more details, see: https://github.com/docker/buildx/blob/master/README.md
+.ensure-docker-buildx:
+	@if docker buildx version > /dev/null 2>&1 || DOCKER_CLI_EXPERIMENTAL="${DOCKER_CLI_EXPERIMENTAL}" docker buildx version > /dev/null 2>&1 ; then \
+	  echo "'docker buildx' is available and enabled."; \
+	else \
+	  echo "installing docker buildx"; \
+	  mkdir -p ~/.docker/cli-plugins; \
+	  curl -sL --output ~/.docker/cli-plugins/docker-buildx https://github.com/docker/buildx/releases/download/v0.4.2/buildx-v0.4.2.$$(go env GOOS)-$$(go env GOARCH); \
+	  chmod a+x ~/.docker/cli-plugins/docker-buildx; \
+	  docker buildx version; \
+	  echo "'docker buildx' is available and enabled. Set DOCKER_CLI_EXPERIMENTAL=enabled if you want to use it."; \
+	fi
```

Comment:
`Docker@19.03.06` on Ubuntu 18.04 is not includes buildx too.

But could force install cli-plugin from binary-release
https://github.com/docker/buildx/blob/master/README.md#binary-release